WordPress/src/main/res/values-ro/strings.xml

```diff
@@ -1652,9 +1652,9 @@ Language: ro
     <string name="site_settings_list_editor_input_hint">Introdu un cuvânt sau frază</string>
     <string name="site_settings_list_editor_no_items_text">Niciun element</string>
     <string name="site_settings_learn_more_caption">Poți suprascrie aceste setări pentru fiecare articol.</string>
-    <string name="site_settings_rp_preview3_site">în „Actualizează”</string>
+    <string name="site_settings_rp_preview3_site">în „Actualizează</string>
```

Comment:
It looks like we should keep both or remove both here.

Reply:
Yeah keep both, I think we should only run those on gutenberg strings, to be sure